ui/src/components/sermons/OnlinePageTabs.js

```diff
@@ -26,129 +32,222 @@ const OnlinePageTabs = ({ sermonNotes }) => {
     setNoteId(noteId + 1);
   };
 
-  return (
-    <Tabs
-      isFitted
-      h="100%"
-      onChange={(i) => setTab(i)} overflowY={(tab ? 'auto' : 'hidden')}
-    >
-      <TabList orientation="horizontal">
-        <Tab>Sermon Notes</Tab>
-        <Tab>New to HMCC</Tab>
+  const IgnitePage = () => {
+    return (
+      <Tabs
+        isFitted
+        onChange={(i) => setTab(i)} overflowY={(tab ? 'auto' : 'hidden')}
+      >
+        <TabList orientation="horizontal">
+          <Tab 
+          _selected={{
+            color: "#46B2D3",
+            borderBottom: "2px solid",
+            borderBottomColor: "#46B2D3",
+          }}
+          borderBottomColor="black"
+          color="white">
+            About Ignite 2022</Tab>
+          <Tab
+          _selected={{
+            color: "#FF7461",
+            borderBottom: "2px solid",
+            borderBottomColor: "#FF7461",
+          }}
+          borderBottomColor="black"
+          color="white">
+          Schedule</Tab>
       </TabList>
-      <TabPanels h="100%">
+      <TabPanels>
         <TabPanel h="100%">
-          <Button
-            mb={3}
-            fontWeight="bold"
-            bg="#F1F1F3"
-            color="#0628A3"
-            fontSize="md"
-            onClick={refreshSermonNotes}
-          >
-            <RepeatIcon />
-          </Button>
-          <Box height="90%" paddingBottom={15}>
-            <iframe
-              key={noteId}
-              title="Sermon Notes"
-              src={sermonNotes}
-              width="100%"
-              style={ isMobile ? {
-                height: '100vh',
-              } : {
-                height: '100%'
-              }}
-            ></iframe>
-          </Box>
+          <VStack spacing={3} align="stretch">
+            {/*red box*/}
+            <Box height="100%" bg= "#FF7461" color="white">
```

Comment:
not sure if this was the actual design, but should these boxes have a borderRadius?